### DISCUSSION TITLE
Fine-tuning : does it actually require the original datasets ?

### DISCUSSION THREAD

**Discussion by Ca-ressemble-a-du-fake** [coqui-ai/TTS#1230] [General Q&A] [answered]
Hi,

I am training from scratch a VITS model based on my own dataset which is about 3/4 of an hour (after filtering it as explained in this [tutorial](https://lordoapple.medium.com/cloning-your-voice-cb321908b060). After some hours of training (30k steps) it begins to sound promising.

If I could accelerate this training process (get a model in hours instead of days), I could develop more voices. So I came back to fine-tuning and I wonder : do I really need the original datasets ? Because I don't see this being mentioned in the [official fine-tuning tutorial](https://tts.readthedocs.io/en/latest/finetuning.html) but it was mentioned in gitter ("We never tested Speaker adaptation without at least partial dataset from training. But if you wanna fine-tune with your data only, you should freeze as much of the model as possible to avoid overfiting"). So I am confused.

My understanding of fine-tuning (or "restore_path") is to use the model weights as a starting point to train on a new dataset without requiring the original datasets. Did I understand correctly the fine-tuning process ?

Thanks

**Comment by erogol** [comment, marked as answer]
IF you only finetune with the new dataset, it might start to forget the previous set of voices it has learned. If your intention is to increase the number of speakers, then it is better to merge the dataset and train the model with all the samples. 

If you only want to use it with the new speaker, then you can keep training with only the new dataset.

**Reply by Ca-ressemble-a-du-fake** [reply]
Oh thank you @erogol , you've made it clearer in my mind!

**Reply by jaggukaka** [reply]
Does it apply to new language as well? Somehow when I fine tune with languages which are from different linguistic family the model doesn't converge at all. But If I fine tune with a different dataset but from the same language as the original trained model (en, fr-fr, br-pt), the training converges pretty fast with the voices from new dataset.

Any insights on this? why fine tuning is not working with a new language? 
I tried what @erogol has mentioned too, including the original datasets for fine tuning along with my new dataset which is of a different language, for some reason training doesn't converge and the model actually unlearns what it has already learned.

**Reply by davidmartinrius** [reply]
Did you figure out how to solve it?
